IonicPortals/IonicPortals/PortalsPubSub.swift

```diff
@@ -11,7 +11,7 @@ import Capacitor
 
 /// An interface that enables marshalling data to and from a ``Portal`` over an event bus
 public enum PortalsPubSub {
-    private static let queue = DispatchQueue(label: "io.ionic.portals.pubsub", attributes: .concurrent)
+    private static let queue = DispatchQueue(label: "io.ionic.portals.pubsub")
```

Comment:
This is now a serial queue because we want to ensure messages are delivered in order